src/Cake.Core.Tests/Cake.Core.Tests.csproj.orig

```diff
@@ -0,0 +1,203 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
@patriksvensson I think this .orig file was included in the check-in, when it shouldn't have been.  Can you removed it?